Makefile

```diff
@@ -72,7 +76,17 @@ drush:
 	$(call message,Executing \"drush -r /var/www/html/web $(COMMAND_ARGS) --yes\")
 	$(call docker-www-data, php drush -r /var/www/html/web $(COMMAND_ARGS) --yes)
 
-prepare: | up
+configuring_env:
+    # If OS Linux - change PHP_TAG in environment.
+	@if [ $(OS_NAME) = "Linux" ]; then \
+		echo "${cyan}${bold}$(PROJECT_NAME): Defined OS LINUX. Change PHP_TAG for docker image in environment${reset}" \
+		# Uncomment all the strings containing 'PHP_TAG'. \
+		sed -i '/PHP_TAG/s/^# //g' \.env; \
+        # Comment all the strings containing 'PHP_TAG' and '-dev-macos-. \
+        sed -i -E '/PHP_TAG.+-dev-macos-/s/^/# /g' \.env; \
+	fi
+
+prepare: | configuring_env | up
```

Comment:
There should not be two `|` symbols in this line, only the first one makes sense.